media/CircleEditor/view-sidebar.js

```diff
@@ -1456,30 +1438,297 @@ sidebar.ArgumentView = class {
+                        for (let j = 0; j < 4; j++) {
+                            original[i * 4 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 5:  // string
+                        break;
+                    case 6:  // bool
+                        if (bufferArr[i] === 'false' || bufferArr[i] - 0 === 0) {
+                            original[i] = 0;
+                        } else {
+                            original[i] = 1;
+                        }
+                        break;
+                    case 7:  // int16
+                        view.setInt16(0, parseInt(bufferArr[i]), true);
+                        for (let j = 0; j < 2; j++) {
+                            original[i * 2 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 8:
+                        break;
+                    case 9:  // int8
+                        view.setInt8(0, parseInt(bufferArr[i]), true);
+                        original[i] = view.getUint8(0);
+                        break;
+                    case 10:  // float64
+                        view.setFloat64(0, parseFloat(bufferArr[i]), true);
+                        for (let j = 0; j < 8; j++) {
+                            original[i * 8 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 11:
+                        break;
+                    case 12:  // uint64
+                        view.setBigUint64(0, BigInt(parseInt(bufferArr[i])), true);
+                        for (let j = 0; j < 8; j++) {
+                            original[i * 8 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 13:
+                        break;
+                    case 14:
+                        break;
+                    case 15:  // uint32
+                        view.setUint32(0, parseInt(bufferArr[i]), true);
+                        for (let j = 0; j < 4; j++) {
+                            original[i * 4 + j] = view.getUint8(j);
+                        }
+                        break;
+                }
+            }
+        }
+
+        const newBuffer = [];
+
+        for (let i = 0; i < Object.keys(original).length; i++) {
+            newBuffer.push(original[i]);
+        }
+
+        this._editObject._arguments._initializer._data = newBuffer;
+    }
+
+    removeBracket(buffer) {
+        let str = buffer;
+        str = str.replace(/\[/g, '');
+        str = str.replace(/\]/g, '');
+        str = str.replace(/\n/g, '');
+        str = str.replace(/ /g, '');
+        str = str.replace(/\{/g, '');
+        str = str.replace(/\}/g, '');
+        str = str.replace(/"/g, '');
+        str = str.replace(/low | high/g, '');
+        const arr = str.split(',');
+        return arr;
+    }
+
+    validationCheck(modified, shape, type) {
```

Comment:
Function name does not match with return values because this function returns modified array if success.
Furthermore, return type of this function is not consistent.